zubhub_backend/zubhub/creators/views.py

```diff
@@ -188,6 +188,7 @@ class RegisterCreatorAPIView(RegisterView):
             "location": "string",\n
             "bio": "",\n
             "subscribe": false\n
+            "creator_tags": ["tag1", "tag2", ...]\n
```

Comment:
yeaa this is good. If we attend to the other reviews above then we'd need to modify this too